source/_integrations/twentemilieu.markdown

```diff
@@ -92,50 +79,50 @@ automation:
         event: start
         entity_id: calendar.twente_milieu
         offset: "-6:00:00"
+        # This triggers 6 hours before the calendar event starts
 
     actions:
       - action: notify.mobile_app_your_device
         data:
           title: "Garbage day!"
-          message: > 
+          message: >
             Reminder: Tomorrow is {{ trigger.calendar_event.summary }} pickup
             day. Don't forget to put out the bin!
 ```
 
-### Send notification at the end of day to haul in the empty bin again
+### Send notification at the end of day to bring in the empty bin
 
-The following example sends a notification to your mobile device at the end of
-the day to remind you to haul in the empty bin again.
+This example sends a notification to remind you to bring the empty bin back in after collection.
 
 ```yaml
 automation:
-  - alias: "Reminder to haul in the bin"
+  - alias: "Reminder to bring in the bin"
     triggers:
       - trigger: calendar
-        event: end 
+        event: end
         entity_id: calendar.twente_milieu
         offset: "-4:00:00"
+        # This triggers 4 hours before the calendar event ends
 
     actions:
       - action: notify.mobile_app_your_device
         data:
-          title: "Haul in the bin!"
-          message: > 
-            Reminder Garbage has been picked up today. Don't forget to haul in
-            the bin!
+          title: "Bring in the bin!"
+          message: >
+            Reminder: The waste has been collected today. Don't forget to
+            bring in your empty bin!
 ```
 
 ## Known limitations
 
-There are no known limitations for this integration.
+- Home Assistant currently doesn't support translating calendar items. Therefore, waste collection events in the calendar will always be displayed in English, regardless of your language settings.
```

Comment:
Is this a known limitation inherit to twente milieu or home assistant?

Reply:
It is an Home Assistant limitation; but in this case, it are fixed items (like Organic waste as an event item), so it pops up as a question quite a lot.